core/components/atoms/avatar/Avatar.tsx

```diff
@@ -61,13 +65,22 @@ export const Avatar = (props: AvatarProps) => {
   const AvatarClassNames = classNames(
     {
       Avatar: true,
-      [`Avatar--${size}`]: size,
+      ['Avatar-square']: shape === 'square',
```

Comment:
Please follow the BEM convention for css classes. Classname here should be like `Avatar--square` as square as a shape works like a modifier here.